packages/www/pages/dashboard/streams/[id]/health.tsx

```diff
@@ -1,148 +0,0 @@
-import React, { useCallback } from "react";
```

Comment:
Is there a way to still keep this as a separate file, even if not a full `page` anymore? Could be a component that is included by the stream details page.

Just feels like they currently have separate logics and were already kept separate, and this would be merging all of them into a single thing, with no clear separation on the purpose of the code.